config/tsconfig.plugin.json

```diff
@@ -1,10 +1,17 @@
 {
   "compilerOptions": {
     "paths": {
-      "appium": ["../packages/appium"]
+      "appium": ["../packages/appium"],
+      "appium/plugin": ["../packages/appium/plugin"],
+      "@appium/plugin-test-support": ["../packages/plugin-test-support"]
```

Comment:
whyj do we need plugin-test-support stuff here, but not driver-test-support stuff?

Reply:
maybe i don't understand what this file is

Reply:
`tsconfig.plugin.json` is reusable ts config _for plugins_.  therefore we don't care about the driver support module